### ISSUE TITLE
Numbers API search errors if no numbers available

### ISSUE THREAD

**Issue by marklewin** [Nexmo/nexmo-php#178]
If there are no numbers available for purchase according my search criteria, I get an error instead of an empty response or meaningful message.

For example, searching for landlines with the `SMS` feature (there should be none):

```php
$response = $client->numbers()->searchAvailable(
    "GB",
    [
        "type" => "landline",
        "features" => "SMS"
    ]
);
```

... results in:

```
Fatal error: Uncaught Nexmo\Client\Exception\Request: number not found in .../vendor/nexmo/client/src/Numbers/Client.php:174
```

**Comment by mheap** [comment]
I've confirmed that this is the existing behaviour and agree that it is incorrect. However, changing the behaviour would be a backwards compatibility break. In the next major version I'd like to return an empty array here.

@lornajane One to fix in the 2.0 release?

**Comment by jaikdean** [comment]
This is closed by #184, right?

**Comment by lornajane** [comment]
Yes but that change is only in the `v2-alpha` branch, not in master yet, so this issue will still exist for most users and I wanted to leave the issue open in case anyone was looking for information about it.

**Comment by dragonmantank** [comment]
This has been merged into the 2.0.0 release.
